Server/src/cli/commands/ui.py

```
@@ -0,0 +1,228 @@
+    "--text", "-t",
+    default="New Text",
+    help="Initial text content."
+)
+@click.option(
+    "--position",
+    nargs=2,
+    type=float,
+    default=(0, 0),
+    help="Anchored position X Y."
+)
+def create_text(name: str, parent: str, text: str, position: tuple):
```

Ruff (0.14.10)

Unused function argument: `position`

(ARG001)